### PR TITLE
VegaLite - LiveBook inspired theme

### PR DESCRIPTION
**Pull request by cristineguadelupe**
--WIP--

CleanShot 2022-04-15 at 01 20 39@2x

CleanShot 2022-04-15 at 01 05 48@2x

### CODE REVIEW

File: lib/assets/vega_lite/main.js [hunk 1]
```diff
@@ -7,7 +7,82 @@ const DEFAULT_DATASET_NAME = "source_0";
...
 const throttledResize = throttle((view) => view.resize(), 1_000);
 
+const markColor = "#6583ff";
+const mainColors = ["#6583ff", "#ffa83f", "#e2474d", "#4aa148", "#2d4cdb", "#b7641f", "#bc1227", "#137518"];
+const blues = ["#b2c1ff", "#8ba2ff", "#6583ff", "#3E64ff", "#2d4cdb", "#1f37b7", "#132593"];
+const lightGray = "#e1e8f0";
+const mediumGray = "#445668";
+const darkGray = "#1c2a3a";
+
+const liveBookTheme = {
```
Comment (jonatanklosko) [inline lib/assets/vega_lite/main.js:17]:
```suggestion
const livebookTheme = {
```

File: lib/assets/vega_lite/main.js [hunk 2]
```diff
@@ -7,7 +7,82 @@ const DEFAULT_DATASET_NAME = "source_0";
...
+  area: {fill: markColor},
+  line: {stroke: markColor, strokeWidth: 2},
+  path: {stroke: markColor},
+  rect: {fill: markColor},
+  shape: {stroke: markColor},
+  symbol: {fill: markColor, strokeWidth: 1.5, size: 50},
+  point: {stroke: markColor},
+  bar: {fill: markColor, stroke: null},
+  circle: {fill: markColor},
+  tick: {fill: markColor},
```
Comment (jonatanklosko) [inline lib/assets/vega_lite/main.js:39]: We may be missing some marks here, like `rule`. Some marks like `boxplot` are composed of base marks, so maybe the theme will apply, but please check :)

Sidenote: there is `Vl.from_json("...")`, which makes it simple to try examples from vega-lite docs.

**Comment by jonatanklosko** [comment]
Looks amazing!!

File: lib/assets/vega_lite/main.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#root {
+  font-family: "Inter";
```
Comment (jonatanklosko) [inline lib/assets/vega_lite/main.css:2]: We don't have any text outside the graph, no?

File: lib/assets/vega_lite/main.js [hunk 4]
```diff
@@ -7,7 +7,85 @@ const DEFAULT_DATASET_NAME = "source_0";
 
 const throttledResize = throttle((view) => view.resize(), 1_000);
 
+const markColor = "#6583ff";
+const mainColors = ["#6583ff", "#ffa83f", "#e2474d", "#4aa148", "#2d4cdb", "#b7641f", "#bc1227", "#137518"];
+const blues = ["#b2c1ff", "#8ba2ff", "#6583ff", "#3E64ff", "#2d4cdb", "#1f37b7", "#132593"];
+const lightGray = "#e1e8f0";
+const mediumGray = "#445668";
+const darkGray = "#1c2a3a";
+const darkestGray = "#0d1829"
```
Comment (jonatanklosko) [inline lib/assets/vega_lite/main.js:16]: Maybe name them with the gray scale we use, like `gray300`. Alternatively an object `const gray = { '300': ...}` :)

File: lib/assets/vega_lite/main.js [hunk 5]
```diff
@@ -7,13 +7,48 @@ const DEFAULT_DATASET_NAME = "source_0";
...
+  "green500": "#4aa148",
+  "green700": "#137518",
+  "gray200": "#e1e8f0",
+  "gray600": "#445668",
+  "gray800": "#1c2a3a",
+  "gray900": "#0d1829",
+};
+
+const primaryColors = [
+  color["blue500"],
```
Comment (jonatanklosko) [inline lib/assets/vega_lite/main.js:31]: Alternatively

```suggestion
  color.blue500,
```

**Review by jonatanklosko — APPROVED** [review]
Just one nitpick, this is fantastic :cat:

File: lib/assets/vega_lite/main.js (hunk 5 above)
Comment (cristineguadelupe) [reply lib/assets/vega_lite/main.js:31]: Done! Let's make you happy! 😛

**Pull request merged**